src/multi_vector_simulator/utils/data_parser.py

```diff
@@ -65,6 +69,8 @@
     DATA,
 )
 
+from multi_vector_simulator.A1_csv_to_json import MissingParameterError
```

Comment:
should we define those in utils rather than in `multi_vector_simulator.A1_csv_to_json` @smartie2076, @SabineHaas ?

Reply:
You mean the different error types? Would probably be good to define all of them at one place, to also have similar error types and not create a lot of them...?

Reply:
They are defined in A1_csv_to_json for the moment, I think it is the only place where they are defined

Reply:
Nope, I also defined personalized error types. Not sure where though..

Reply:
Ok, then utils would be a good place to define them I think

Reply:
Is this for a later PR? Then, do you want to create an issue for it?

Reply:
This PR is small enough to add it here